cli/deploy-managed-online-endpoint-access-resource-uai.sh

```diff
@@ -54,41 +54,44 @@ az storage blob upload --account-name $STORAGE_ACCOUNT_NAME --container-name $ST
 
 # <get_user_identity_client_id>
 uai_clientid=`az identity list --query "[?name=='$UAI_NAME'].clientId" -o tsv`
+uai_principalid=`az identity list --query "[?name=='$UAI_NAME'].principalId" -o tsv`
+uai_type=`az identity list --query "[?name=='$UAI_NAME'].type" -o tsv`
```

Comment:
I don't see this being used anywhere. Pls remove if this is not required as it will get rendered in public docs: https://docs.microsoft.com/en-us/azure/machine-learning/how-to-deploy-with-rest